pyfr/backends/openmp/base.py

```diff
@@ -41,8 +43,7 @@ def __init__(self, cfg):
         self._providers = [k(self) for k in kprovcls]
 
         # Instantiate optional kernel provider classes
-        for k in [xsmm.OpenMPXSMMKernels, gimmik.OpenMPGiMMiKKernels,
-                  cblas.OpenMPCBLASKernels]:
+        for k in [xsmm.OpenMPXSMMKernels, gimmik.OpenMPGiMMiKKernels]:
```

Comment:
Looks as if the only optional provider now is XSMM (GiMMiK is a hard dependency).  No need for the loop.